pages/tidal tools/analyze_database.md

```diff
@@ -19,14 +19,14 @@ It is capable of analyzing both **Oracle and SQL Server databases**. Providing a
 
 Tidal Tools is able to analyze Oracle and SQL Server databases with the following versions:
 
-| Oracle                | SQL Server |
-|-----------------------|
-| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 8i Database Analysis) Oracle Database 8i (8.1) | SQL Server 2008R2 |
-| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 9 Database Analysis) Oracle Database 9i Release 2 (9.2)   | SQL Server 2016 |
-| Oracle Database 10g Release 2 (10.2) | SQL Server 2017 |
-| Oracle Database 11g Release 1 (11.1) |
-| Oracle Database 11g Release 2 (11.2) |
-| Oracle Database 12c Release 1 (12.1) |
+| Oracle                | SQL Server | MySQL | PostgreSQL |
+|-----------------------|------------|-------|------------|
+| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 8i Database Analysis) Oracle Database 8i (8.1) | SQL Server 2008R2 | 5.6 | 8 |
```

Comment:
@pshevtsov I think there is something off with the hyperlink here.

![image](https://user-images.githubusercontent.com/22829270/110146313-cbb0a680-7db0-11eb-92fc-68c3375f693d.png)

Reply:
It renders OK with Jekyll
![image](https://user-images.githubusercontent.com/290451/110150305-29181800-7df0-11eb-9227-b2a29074cec4.png)

Reply:
oh, great then. 👍  🚀